crypto/perlasm/x86_64-xlate.pl

```diff
@@ -115,7 +115,12 @@
 	.long 4f - 1f
 	.long 5
 0:
-	.asciz "GNU"
+	# Replace .asciz "GNU" with .byte since .asciz isn't supported
+	# on Solaris.
```

Comment:
I understand that the comment is spur of the moment, but without the history, it looks odd.  May I suggest this?

```
	# "GNU" encoded with .byte, since .asciz isn't supported
	# on Solaris.

Reply:
Approval stands with comment change. Good idea @levitte.

Reply:
Done.